app.py

```diff
@@ -1,57 +1,23 @@
+
 import json
 import os
 import time
 
-import numpy as np
 from packaging import version  # not built-in, need pip install
 
 
-def input_numpy(array: 'np.ndarray', axis: int = 0, size: int = None, shuffle: bool = False):
-    """This function is in Jina since very early version, but it was at different places due to refactoring.
-    so I copy-paste here for compatibility
-    """
-    if shuffle:
-        # shuffle for random query
-        array = np.take(array, np.random.permutation(array.shape[0]), axis=axis)
-    d = 0
-    for r in array:
-        yield r
-        d += 1
-        if size is not None and d >= size:
-            break
-
-
-def input_fn(fp, index=True, num_doc=None):
-    """This function is used before v0.2.2"""
-    from jina.helloworld.helper import load_mnist
-    img_data = load_mnist(fp)
-    if not index:
-        # shuffle for random query
-        img_data = np.take(img_data, np.random.permutation(img_data.shape[0]), axis=0)
-    d_id = 0
-    for r in img_data:
-        yield r.tobytes()
-        d_id += 1
-        if num_doc is not None and d_id > num_doc:
-            break
-
-
 def benchmark():
     try:
-        # only these three imports are immutable from 0.1 upto now
-        from jina import __version__
-        from jina.flow import Flow
-        try: 
-            from jina.helloworld.fashion.helper import load_mnist
-        except:
-            from jina.helloworld.helper import load_mnist
-
+        from jina import __version__, Flow
+        from jina.helloworld.fashion.helper import load_mnist
+        from jina.helloworld.fashion.executors import MyEncoder, MyIndexer
+        from jina.types.document.generators import from_ndarray
 
         from pkg_resources import resource_filename
 
         err_msg = ''
         index_size = 60000
-        query_size = 4096
+        query_size = 10000
```

Comment:
![image](https://user-images.githubusercontent.com/32271197/119775609-6d4e1c80-bef6-11eb-882f-e0d408981bcb.png)
I find that the `query_size` may be 10000

Reply:
```python
'index_time': index_time,
'query_time': query_time,
'index_qps': index_size / index_time,
'query_qps': query_size / query_time,
```
`index_size` is the number of the whole index data, but `query_size` is not.